x/gateway/client/cli/tx_stake_gateway.go

```diff
@@ -3,29 +3,35 @@ package cli
 import (
 	"strconv"
 
+	"pocket/x/gateway/types"
+
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/client/flags"
 	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/spf13/cobra"
-	"pocket/x/gateway/types"
 )
 
 var _ = strconv.Itoa(0)
 
 func CmdStakeGateway() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "stake-gateway",
+		Use:   "stake-gateway [amount]",
 		Short: "Broadcast message stake-gateway",
-		Args:  cobra.ExactArgs(0),
+		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-
 			clientCtx, err := client.GetClientTxContext(cmd)
 			if err != nil {
 				return err
 			}
-
+			stakeAmountString := args[0]
```

Comment:
s/stakeAmountString/stakeString
s/stakeAmount/stake

It includes both amount & denom